src/test/resources/log4j2.xml

```diff
@@ -38,6 +30,14 @@
             <AppenderRef ref="File"/>
         </Logger>
 
+        <!-- Relevant classes we want to see INFO logs on the Console -->
+        <Logger name="com.salesforce.storm.spout.dynamic">
```

Comment:
@Crim note the change here, explicitly moving the INFO to the file rather than the console. Should make things less noisy in Travis.

Reply:
Is the comment incorrect now and need to updated to match?

Reply:
Good catch.

Reply:
Done.